libraries/c_sdk/standard/mqtt/include/iot_mqtt.h

```diff
@@ -1,5 +1,5 @@
 /*
- * FreeRTOS MQTT V2.1.1
+ * FreeRTOS MQTT V2.1.1 SHIM
```

Comment:
Since the Shim is getting introduced for the first time, I think it will be of version 1.0.0. Also, since this PR is about syncing changes, I would have made it a different small PR to handle just that.